spec/models/video_spec.rb

```diff
@@ -4,4 +4,33 @@
   it {should belong_to (:category)}
   it {should validate_presence_of (:title)}
   it {should validate_presence_of (:description)}
+
+  describe "search_by_title" do
```

Comment:
In RSpec describe messages, its always better to label these with prepending `.` or `#`.

`.` means that you are going to work on class method. and `#` would be instance method.

Read more about it here -> http://betterspecs.org/#describe